content/en/service_management/service_level_objectives/monitor.md

```diff
@@ -142,3 +132,4 @@ Confirm you are using the preferred SLI type for your use case. Datadog supports
 [3]: /service_management/service_level_objectives/metric/
 [4]: /synthetics/api_tests/?tab=httptest#alert-conditions
 [5]: /service_management/service_level_objectives/#slo-status-corrections
+[6]: /service_management/service_level_objectives/time_slice/
```

Comment:
Sorry if I missed this, I didn't see the associated reference link in the docs.

Reply:
@estherk15 can you clarify where/how to add the reference link?

Reply:
@roxanne-moslehi If you want to hyperlink to this page, you can add brackets around the word like `[Time Slice SLO][6]`

Reply:
@estherk15 Oh yeah I see, I had added a hyperlink but realized it didn't work in the "alert" added at the top of the page